libraries/botbuilder-ai/.eslintrc.json

```diff
@@ -1,3 +1,4 @@
 {
-    "extends": "../../.eslintrc.json"
-}
\ No newline at end of file
+    "extends": "../../.eslintrc.json",
+    "plugins": ["only-warn"]
```

Comment:
Why are we disabling lint errors? I fixed existing errors and explicitly disabled `only-warn` in https://github.com/microsoft/botbuilder-js/pull/3225. New errors should be fixed, not suppressed 😄